templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
```

Comment:
Why is there a separate form just for the drag and drop uploading? Should this be incorporated into the form that already exists on this page?